core/modules/admin_bar/css/admin_bar.css

```diff
@@ -168,8 +168,9 @@
   padding-right: 10px;
   padding-left: 28px;
 }
-/* Override the margin added to the 'layout' class */
+/* Override '.layout' class styling */
```

Comment:
Since we're updating this inline comment, we might as well add the missing period (coding standards please).

Reply:
Fixed.